src/translations/utils.ts

```diff
@@ -0,0 +1,18 @@
+import merge from 'lodash.merge';
+import {AppOrgs} from '../../types/app-orgs';
+import {APP_ORG} from '@env';
+
+type Overrides<T> = {
+  [P in keyof T]?: Overrides<T[P]>;
+};
+export default function orgSpecificTranslations<T>(
+  translationTexts: T,
+  overrides: Partial<{[org in AppOrgs]: Overrides<T>}>,
+) {
+  const test = {...translationTexts, ...overrides.nfk};
+  if (APP_ORG === AppOrgs.nfk) {
+    return merge(translationTexts, overrides.nfk);
+  } else {
+    return merge(translationTexts, overrides.atb);
+  }
```

Comment:
Vil vi ikke kunne gjøre direkte
```suggestion
  return merge(translationTexts, overrides[APP_ORG]);
```

Og slik sett automatisk håndtere alle AppOrgs?

Reply:
Jo. Så lenge APP_ORG er definert, men er ikke noe som sjekker at env-variabler faktisk blir satt. Så burde egentlig legge til noen sjekker på det.

Reply:
Det blir vel fort den her? https://github.com/AtB-AS/mittatb-app/issues/2034

Reply:
Lurer på om det går an å skru på at env ikke kan være undefined. https://www.npmjs.com/package/react-native-dotenv/v/2.5.1

Reply:
Blir runtime feil da

Reply:
Tror vi bør validere det build-time ja, og det blir fort CI

Reply:
Og akkurat for denne koden, så vil udefinert type føre til undefined som merges inn med translations som gjør ingenting. Som er det jeg ville forventet av koden uansett på et vis